linkedlist.py

```diff
@@ -0,0 +1,86 @@
+class Node:
+    def __init__(self, data=None, next=None):
+        self.data = data
+        self.next = next
```

Comment:
dla takich klas ktore sa tylko wraperami na dane, tzn nie zawieraja (prawie) zadnej szczegolnej logiki itp, warto stosowac taki wrapper [dataclass](https://docs.python.org/3/library/dataclasses.html). To sprawi, ze automatycznie wygeneruje ci sie `__init__` i pare innych podstawowych funkcji, jak np porownania `==` itd

```python
from dataclasses import dataclass

@dataclass
class Node:
    data: Any
    next: Optional["Node"]  # tu typ jest w ciapkach, poniewaz jeszcze nie skonczyla sie deklaracja `Node`, wiec jeszcze nie za bardzo mozna go uzyc, to ten 'hack' w pythonie o ktorym ci pisalem
```
![image](https://user-images.githubusercontent.com/34972409/217602669-7d4eece1-54ff-4250-a136-36ff5117a18d.png)